lib/std/ascii.zig

```diff
@@ -288,36 +288,46 @@ test "ascii character classes" {
     testing.expect(isSpace(' '));
 }
 
+/// Writes a lower case copy of `ascii_string` to `output`.
+pub fn lowerString(ascii_string: []const u8, output: []u8) []u8 {
+    for (output) |*c, i| {
+        c.* = toLower(ascii_string[i]);
```

Comment:
This is dangerous when `ascii_string` is shorter than `output`.

Reply:
Thanks. The dangers of copy-paste.